cfgov/v1/jinja2/v1/includes/rich-text/footnote-reference.html

```diff
@@ -0,0 +1 @@
+<a aria-labelledby="footnotes" href="#footnote-{{ index }}" id="footnote-source-{{ index }}"><sup>{{ index }}</sup></a>
```

Comment:
Does it matter that the `aria-labelledby` is for the whole footnotes area and not the specific linked footnote?

Reply:
It's based on https://www.niquette.ca/articles/accessible-footnotes/#descriptive_markers and similar research, which suggests that this will read as "Link 2 Footnotes" rather than just "Link 2". So, my understanding is that `aria-labeledby` will read the destination `id` as the label for the link. 

In my research, it seems that the general title of the footnotes/references/endnotes/etc section is preferable here to a label specific to the note in question.